app/models/manager.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: wyeworkers
+#
+#  id         :integer          not null, primary key
+#  name       :string           not null
+#  type       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Manager < Wyeworker
+  before_create do
+    self.name = self.name || "Anonymous manager"
+  end
```

Comment:
Same as above with the current model data the only identifying quality of the Wyeworker is the name

Reply:
Fair enough!